src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ CHANGELOG
  * Add support for union types with `Symfony\Component\EventDispatcher\Attribute\AsEventListener`
  * Add `framework.allowed_http_method_override` option
  * Initialize `router.request_context`'s `_locale` parameter to `%kernel.default_locale%`
+ * Add `use_sapi_error_renderer` option to use the appropriate error renderer based on the `PHP_SAPI`
```

Comment:
Other changelog entries have the `framework.` prefix. I find it easier to understand when you're referring to the bundle configuration.

```suggestion
 * Add `framework.use_sapi_error_renderer` option to use the appropriate error renderer based on the `PHP_SAPI`
```